src/shared/bus-util.h

```diff
@@ -49,9 +49,7 @@ int bus_connect_transport_systemd(BusTransport transport, const char *host, Runt
 
 int bus_log_address_error(int r, BusTransport transport);
 int bus_log_connect_full(int log_level, int r, BusTransport transport, RuntimeScope scope);
-static inline int bus_log_connect_error(int r, BusTransport transport, RuntimeScope scope) {
-        return bus_log_connect_full(LOG_ERR, r, transport, scope);
-}
+int bus_log_connect_error(int r, BusTransport transport, RuntimeScope scope);
 
 #define bus_log_parse_error(r)                                  \
         log_error_errno(r, "Failed to parse bus message: %m")
```

Comment:
By the way, WDYT about calling logger through macro like that?
With this commit, the macro does not work anymore unless also log.h is included.

So, I think the header should include log.h, unless that causes any conflicts.